examples/llm_ptq/hf_ptq.py

```diff
@@ -504,6 +504,12 @@ def main(args):
             quant_cfg["quant_cfg"]["*radio*"] = {"enable": False}
             quant_cfg["quant_cfg"]["*visual*"] = {"enable": False}
 
+        if model_type == "qwen3next" and args.qformat == "nvfp4":
+            quant_cfg["quant_cfg"]["model*.*attn*in_proj*"] = {"enable": False}
+            quant_cfg["quant_cfg"]["model*.*attn*q_proj*"] = {"enable": False}
+            quant_cfg["quant_cfg"]["model*.*attn*k_proj*"] = {"enable": False}
+            quant_cfg["quant_cfg"]["model*.*attn*v_proj*"] = {"enable": False}
+
```

Comment:
should this change be added to [multinode_ptq.py](https://github.com/NVIDIA/Model-Optimizer/blob/d650ad556b3a4ba330aa7416d4a27cce3f70ac69/examples/llm_ptq/multinode_ptq.py)" as well?